src/server/aws/batchDeleteHandler.spec.ts

```diff
@@ -54,7 +55,7 @@ describe('batchDeleteHandler', () => {
     await batchDeleteHandler.pollQueue();
     expect(
       sentryStub.calledOnceWithExactly(error, {
-        level: Sentry.Severity.Critical,
+        level: 'fatal' as SeverityLevel,
```

Comment:
"Critical" no longer exists as an option - it's "Fatal" now.

Reply:
hmm wondering why the type cast? Can we import the `fatal` type from `@sentry/types`?